src/dataflow/sink/kafka.rs

```diff
@@ -6,8 +6,12 @@
 // As of the Change Date specified in that file, in accordance with
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
+use futures::executor::block_on;
```

Comment:
`futures::executor::block_on` should go in the next block. Please also preserve the gap between the license comment and the `use` block.